angular/src/environments/environment.ts

```diff
@@ -1,10 +1,10 @@
 export const environment = {
   production: false,
   fusionauth: {
-    apiUrl: 'http://localhost:9011',
-    applicationId: 'ad020c90-e0c8-4173-b914-d1409b9a5da9'
+    apiUrl: "http://localhost:9011",
+    applicationId: "bf499692-4847-440d-b46f-c264113890ea	"
```

Comment:
Is this right padded on purpose? FYI, this application Id will always be dev / instance specific.

Reply:
Nope, that was not intentional. I can correct that in a future PR.

Yeah, we may want to just replace these with placeholders, such as `YOUR_APPLICATION_ID_HERE`

Reply:
Great, thanks. Yeah, probably the same for API key, client secret, client id, but I'm fine with just documenting they will need to be changed. Use your judgement.